src/main.scss

```diff
@@ -67,48 +132,713 @@ header {
 				}
 			}
 		}
-		@media only screen and (max-width: 480px) {
-				display: none;
-		}
 	}
 }
 
-        // nav 2 //
-.nav-2 {
+// nav 2 //
+.nav-2 {	
 	background-color: $nav-2-color;
-	.left-buttons{
-		.button-1{
-			img{
-				height: 32px;
+	height: 100px;
+	@media (max-width: $screen-md-min){
+		height: 210px;
+	}
+	@media (max-width: $screen-xs-min){
+		height: 70px;
+	}
+	.container {
+		height: 100%;
+		.l-side-gray-nav {
+			padding-left: 0;
+			@media (max-width: $screen-xs-min) { 
+				padding: 0
 			}
+			
+				.left-buttons {
+					padding-left: 0;
+					@media (max-width: $screen-xs-min) { 
+						width: 100%;
+						padding: 0
+					}
+					.button {
+						direction: rtl;
```

Comment:
Since this site should support both Hebrew and English you should just set the `direction: rtl;` property on the `body` element and remove it from everywhere else. This way when the `eng` button is clicked you can just change it on the body to be `ltr`.

Note! When you do this change some stuff may swap in direction. Don't panic, take a deep breath, you just need to swap the order in the HTML and everything will be fine. I would commit everything before doing this chage so you work on a clean slate.